firefox-ios/Client/Helpers/MenuBuilderHelper.swift

```diff
@@ -67,7 +74,13 @@ class MenuBuilderHelper {
             ]
         )
         fileMenu.children.forEach {
-            ($0 as! UIKeyCommand).wantsPriorityOverSystemBehavior = true
+            guard let fileMenuKeyCommand = $0 as? UIKeyCommand else {
+                logger.log("Failed to cast file menu child to UIKeyCommand in MenuBuilderHelper class",
+                           level: .fatal,
```

Comment:
i'd log those as a `.info` level since there is no crash there and we don't send them to sentry as well